dap-hydra.el

```diff
@@ -65,8 +67,9 @@ _Q_: Disconnect     _sS_: List sessions      _bl_: Set log message   _eis_: Insp
   ("eii" dap-ui-inspect)
   ("eir" dap-ui-inspect-region)
   ("eis" dap-ui-inspect-thing-at-point)
-  ("q" nil "quit" :color blue)
```

Comment:
What do you think about:

"dd" -> dap-debug
"dr" -> dap-debug-recent
"dl" -> dap-debug-last
"dt" -> dap-debug-edit-template.

Reply:
Fine.

Reply:
Did anything come of this? Its not in the hydra as of today.

Reply:
I've addressed @yyoncho 's comment and re-done the PR: https://github.com/emacs-lsp/dap-mode/pull/341